.travis.yml

```diff
@@ -26,6 +26,7 @@ before_script:
   - cp config/application.yml.example config/application.yml
   - cp certs/saml.crt.example certs/saml.crt
   - cp keys/saml.key.enc.example keys/saml.key.enc
+  - cp keys/pii_signing.key.enc.example keys/pii_signing.key.enc
```

Comment:
I hear that. Could refactoring scope we touch be considered always in scope? I.e., Red, green, refactor?